src/common/geo.ts

```diff
@@ -264,14 +266,30 @@ export const countryCodeToContinent: Record<string, Continent> = {
 
 let geoReader: ReaderModel | null = null;
 
-export const initGeoReader = async (): Promise<ReaderModel | undefined> => {
+export /**
+ * Loads the GeoIP2 database and returns a reader instance
+ * Requires the GEOIP_PATH environment variable to be set
+ * and file to be mounted, in prod it is mounted as a volume
+ * check .infra for more details
+ *
+ * @return {*}  {(Promise<ReaderModel | undefined>)}
+ */
+const initGeoReader = async (): Promise<ReaderModel | undefined> => {
```

Comment:
Looks weird to have the comment between `export` and `const` 😬

Reply:
thats how document plugin does it 🤷